src/TulsiGenerator/Bazel/tulsi/tulsi_aspects.bzl

```diff
@@ -477,6 +477,13 @@ def _collect_module_maps(target):
       maps += module_maps
   return maps
 
+def _get_module_name(label):
```

Comment:
It's required for indexing objective-c from swift properly as well since objc->swift interop depends on clang modules with bridging headers aren't used. Have to match this logic here.

Also hope it to support the "module_name" property on objc_library if this every gets merged: https://github.com/bazelbuild/bazel/pull/2313

I agree it would be nice for it to be exposed on providers, however it would require changes to the objc_provider as well. It also looks like the objc_provider doesn't have any non-transitive properties I can find.